integration_tests/test_employment.py

```diff
@@ -74,27 +75,18 @@ def test_movers_change_employment(populations):
 
 def test_employment_income_propensity_updates(simulants_on_adjacent_timesteps):
     for before, after in simulants_on_adjacent_timesteps:
-        changed_jobs_idx = before.index[
-            (before["employer_id"] != after["employer_id"])
-            & (np.floor(before["age"]) == np.floor(after["age"]))
-        ]
+        changed_jobs = before["employer_id"] != after["employer_id"]
 
+        # Employer income propensity changes when employer changes.
```

Comment:
I was imagining:

```
changed_employer_propensity = before["employer_income_propensity"] != after["employer_income_propensity"]
assert (changed_jobs == changed_employer_propensity).all()
```
instead of lines 81-90.